include/cutlass/numeric_conversion.h

```diff
@@ -2441,6 +2442,225 @@ struct FastNumericArrayConverter<int8_t, float, N, Round> {
   result_type operator()(source_type const &s) const { return convert(s); }
 };
 
+
+/// Partial specialization for Array<cutlass::half_t, 4> <= Array<int8_t, 4>
+template <FloatRoundStyle Round>
+struct FastNumericArrayConverter<cutlass::half_t, int8_t, 4, Round> {
+  using result_type = Array<cutlass::half_t, 4>;
+  using source_type = Array<int8_t, 4>;
+  static FloatRoundStyle const round_style = Round;
+
+
+  CUTLASS_DEVICE
+  static result_type convert(source_type const &source) {
+    result_type result;
+    
+    #if 0 // Scalar conversion (Please keep this code for reference for vectorized version below)
+    CUTLASS_PRAGMA_UNROLL
+    for (int i = 0; i < 4; ++i) {
+      int16_t tmp = source[i] + 26112 /* 0x6600 */;
+      result[i] = reinterpret_cast<cutlass::half_t const &>(tmp) - 1536.0_hf;
+    }
+    #endif
+
+    // Vectorized s8->f16 conversion using packed instructions
+    uint32_t const* source_ptr = reinterpret_cast<uint32_t const*>(&source);
+    uint32_t* result_ptr = reinterpret_cast<uint32_t*>(&result);
+
+    // Pack s8x2 (s8[1], s8[0]) -> s16x2 (sext.s8[1], sext.s8[0])
+    // (See https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#data-movement-and-conversion-instructions-prmt)
+    // The inline ptx below uses `msb=0` and `msb=1` from the above link to sext the sign-bit in 0, 1, 2, 3 bytes of s8x4
+    // into result_ptr[0] and result_ptr[1]'s 08-15 and 24-31 bits, respectively.
+    // Note that `__byte_perm(source_ptr[0], source_ptr[0], 0x9180);` won't acheive the same and doesn't sext the sign-bit.
+    // Thus, we use inline ptx `prmt.b32` instruction for the desired sext of s8x2 to s16x2.
```

Comment:
```suggestion
    // Note that `__byte_perm(source_ptr[0], source_ptr[0], 0x9180);` won't acheive the same and doesn't set the sign-bit.
    // Thus, we use inline ptx `prmt.b32` instruction for the desired set of s8x2 to s16x2.
```
Typo? Or is it short for sign extension/extend?

Reply:
sext = sign-extend here.